package.json

```diff
@@ -20,16 +20,46 @@
     "Other"
   ],
   "activationEvents": [
-    "onCommand:codechecker.helloWorld"
+    "onCommand:codechecker.backend.reloadMetadata",
+    "onView:codechecker.views.overview"
   ],
   "main": "./dist/extension.js",
   "contributes": {
     "commands": [
       {
-        "command": "codechecker.helloWorld",
-        "title": "Hello World"
+        "command": "codechecker.backend.reloadMetadata",
```

Comment:
Please extend the main readme file with the commands and write a few words about them.

Also please add a section in the main readme file what is the recommended way to try out / install this plugin.

Reply:
Added a basic README, I'll expand it with animations when there's enough features for it.